### PR TITLE
feat: experience view updates

### PR DESCRIPTION
**Pull request by dubbelnisse**
**Ticket:** [#181](https://trello.com/c/YIBIzFxL/181-fixar-f%C3%B6r-erfarenhet-2)
**Intent:** Updates based on designer feedback.
**How to test (optional):** Check comments in Trello and compare this branch towards CI environment.

### All of the following commands should pass.

- [x] `npm test`
- [x] `npm run lint`
- [ ] `npm run cypress`

### Browser testing

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] IE11
- [ ] Edge

### CODE REVIEW

**Comment by lundeeell** [comment]
Also, TimeLineInfo should have size xsmall, not small 😄

**Comment by lundeeell** [comment]
And maybe att more margins when between timeline and editing 
image

File: src/components/Layout/RegistrationLayout.tsx [hunk 1]
```diff
@@ -77,7 +79,7 @@ const RegistrationLayout: React.FC<
         <Paragraph lineHeight="100%">{headerText}</Paragraph>
       </NavigationContainer>
       {children}
-      {nextPath && (
+      {nextPath && !hideNextBtn && (
```
Comment (Jimjardland) [inline src/components/Layout/RegistrationLayout.tsx]: Would probably rename this to `showNextButton` instead, easier to read true than false imo!

Comment (dubbelnisse) [reply src/components/Layout/RegistrationLayout.tsx]: Good catch, I agree.

**Comment by dubbelnisse** [comment]
> When editing an experience on android, UI gets messed up as keyboard only pushes some fields up
> image

**Pull request merged**